SwiftCompilerSources/Sources/SIL/Instruction.swift

```diff
@@ -1231,6 +1245,10 @@ public class TermInst : Instruction {
   }
   
   public var isFunctionExiting: Bool { false }
+
+  public var isSafeNonExitTerminator: Bool {
```

Comment:
I know you just copied it from C++, but we need to do better. You need to specify what "safe" means in this context (it's not clear to me, either).

Also, please don't bridge this property, but define it for all the term inst classes in swift, like `isFunctionExiting`.

Reply:
My bad on the blind copying. This signifies the same thing as the existing `isFunctionExiting` property on terminator instructions.

Reply:
Removed this altogether as the context in which it was needed has been simplified.